### PR TITLE
Move GCP functions to dedicated lib dir

### PR DESCRIPTION
**Pull request by aarontp**
Going to move this as is in one PR, and then I'll add the cloud function specific google_cloud objects in the next.

### CODE REVIEW

File: turbinia/processors/google_cloud.py [hunk 1]
```diff
@@ -24,292 +24,13 @@
 
 from turbinia import config
 from turbinia import TurbiniaException
+from turbinia.lib import google_cloud
```
Comment (berggren) [inline turbinia/processors/google_cloud.py]: Maybe this instead:
from turbinia.lib.google_cloud import GoogleCloudProject

Comment (aarontp) [reply turbinia/processors/google_cloud.py]: Yeah, that's probably better.  Done.

**Comment by aarontp** [comment]
PTAL.

**Comment by berggren** [comment]
LGTM

**Pull request merged**